src/main/java/org/mockbukkit/mockbukkit/block/data/BlockDataKey.java

```diff
@@ -76,6 +76,13 @@ public enum BlockDataKey
 	 * Stores what {@link Bisected.Half} a {@link Bisected} block is placed in.
 	 */
 	HALF("half", string -> Bisected.Half.valueOf(string.toUpperCase(Locale.ROOT)), Bisected.class::isInstance),
+	/**
+	 * Stores what {@link Bisected.Half} a {@link Bisected} block is placed in.
+	 * Similar to HALF but stores the value as lower or upper.
+	 *
+	 * @see BisectedDataMock
+	 */
+	HALF_RAW("half", a -> a, Bisected.class::isInstance),
```

Comment:
An alternative to this is to do this, where isOneBlockStructure is annotaded Internal, and HalfSerdes is a serializer / deserializer. 
```suggestion
	HALF_SINGLE_BLOCK("half", new HalfSerdes(true), object -> object instanceof BisectedMock bisected && bisected.isOneBlockStructure()),
	/**
	 * Stores what {@link Bisected.Half} a {@link Bisected} block is placed in.
	 * Similar to HALF but stores the value as lower or upper.
	 *
	 * @see BisectedDataMock
	 */
	HALF_MULTI_BLOCK("half", new HalfSerdes(false), object -> object instanceof BisectedMock bisected && !bisected.isOneBlockStructure()),
```
I guess this would need a bit more work, but from my point of view, more consistent with the current way things are done. What do you think? Am I over-complicating things as usual?

Reply:
I need to validate how am i going to check the `isOneBlockStructure` because using the classes directly we don't have this type of information yet